srcpkgs/cdb/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'cdb'
 pkgname=cdb
 version=0.75
-revision=4
+revision=5
```

Comment:
Not sure this one needs a revbump.

Reply:
It doesn't, but I've revbumped everything else while doing this so why stop now ;)